src/languages/en.ts

```diff
@@ -3213,6 +3213,13 @@ export default {
     },
     subscription: {
         mobileReducedFunctionalityMessage: 'You can’t make changes to your subscription in the mobile app.',
+        billingBanner: {
+            preTrial: {
+                title: 'Start a free trial',
+                subtitle: 'To get started, ',
+                subtitleLink: 'complete your setup checklist here',
```

Comment:
Sorry for the late comment, but just checking to see if we wanted to have a period at the end of this? cc @jamesdeanexpensify

Reply:
Yes, great catch! After `here` please!

Reply:
> Yes, great catch! After `here` please!

Sure. We will integrate this change during implementation of issue [here](https://github.com/Expensify/App/issues/43674)
cc @fabioh8010